test/mongoShell.test.ts

```diff
@@ -0,0 +1,281 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This is a start at mongo shell tests.  Currently on works on Windows locally (not in pipeline)